File: c/5-Arrays/practice/ap-1.c
#include <stdio.h>
int main()
{
    int a;
    if(scanf("%d",&a)!=1||a<=0)
    {
        printf("Invalid Input");
        return 0;
    }
    
    int arr[a];
    for(int i =0;i<a;i++)
    {
        scanf("%d ",&arr[i]);
    }
    int no_zero =0;
    for(int i =0;i<a;i++)
    {
        if(arr[i]!=0)
        {
            arr[no_zero++]=arr[i];
        }
    }
    while(no_zero<a)
    {
        arr[no_zero++]=0;
    }
    for(int i =0;i<a;i++)
    {
        printf("%d ",arr[i]);
    }
    
    
}

File: java/4-Arrays/practice/ap-1.java
import java.util.Scanner;
public class P
{
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
       if(n<0)
        {
            System.out.println("Invalid input");
            return;
        }
        int[]arr=new int[n];
        int mul=1;
        for(int i=0;i<n;i++)
        {
            if(!sc.hasNextInt())
            {
                System.out.println("Invalid input");
                return;
            }
            arr[i]=sc.nextInt();
            mul*=arr[i];
        }
        System.out.println(mul);
    }
}